docs/hardware/common.md

```diff
@@ -0,0 +1,223 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Hardware Common Metrics
+--->
+
+# Semantic Conventions for Hardware Common Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+All metrics in `hw.` instruments should be attached to a [Host Resource](/docs/resource/host.md)
```

Comment:
Is is a responsibility of instrumentation author or the end user?

Reply:
This is refactoring (copy-paste) from original file.

Reply:
Responsibility is on the instrumentation developer.

Reply:
@lmolkova I'm sorry, but there must be a misunderstanding here. Let's say I'm HPE developing an OpenTelemetry instrumentation for my ProLiant servers, I will use an Otel SDK to push metrics that I captured through specific hardware busses, and while developing this software layer, I will need to create a Resource to attach my metrics to, so that both Resources and Metrics are sent over OTLP.

It's simply a matter of making sure all metrics have the proper attributes to attach them to a host.

Without these, it's practically impossible to use these metrics in real life.

Reply:
@lmolkova I understand. 

For my own knowledge, could you please clarify what you call "instrumentation"?

Reply:
Should I rephrase this sentence then such as 
```suggestion
Application developers should attach all the metrics in the `hw` namespace to a `Host Resource](/docs/resource/host.md)."
```